components/collections/src/codepointinvliststringlist/mod.rs

```diff
@@ -205,6 +207,16 @@ impl<'data> CodePointInversionListAndStringList<'data> {
     pub fn contains_char(&self, ch: char) -> bool {
         self.contains32(ch as u32)
     }
+
+    /// See [`CodePointInversionList::contains_range`]. Ignores strings.
+    pub fn contains_range(&self, range: &impl RangeBounds<char>) -> bool {
+        self.cp_inv_list.contains_range(range)
+    }
+
+    /// See [`CodePointInversionList::contains_set`]. Ignores strings.
+    pub fn contains_char_set(&self, set: &CodePointInversionList) -> bool {
+        self.cp_inv_list.contains_set(set)
+    }
```

Comment:
We could also expose the underlying CodePointInversionList instead, it's not like we're leaking implementation details if it's already in the type name.

Something like `to_code_point_inversion_list(&self) -> CodePointInversionList<'_>`?

Reply:
Yeah, the name of the class ties it to the underlying implementation, so it's fine by me to return it. How about .code_points() ?

Reply:
`code_points()` is better, thanks!